src/components/donationCard/DonationCard.scss

```diff
@@ -0,0 +1,125 @@
+// * {
+//   border: 1px solid red;
+// }
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@300;500&display=swap");
```

Comment:
Maybe good to import the font in one place and use it everywhere?

made a card for it in trello